### ISSUE TITLE
Support for new metadata fields added since 1.0.0 release

### ISSUE THREAD

**Issue by nicktrosnant** [u3a-siteworks-development/u3a-importexport#9]
Need export and import routines extended to cover:
- group start and end times
- event end times
- multiple group categories
and possibly other changes?

**Comment by edwery** [comment]
If we are going to support compound or nested structures, it would be sensible to start serializing as JSON instead of CSV.  There is full support for JSON in PHP, and as long as we are the ones deserializing and serializing, then it should be quite easy to implement.

**Comment by nicktrosnant** [comment]
Yes, but the main reason for using the CSV format is that a user can open the file in a spreadsheet and manually enter data or make changes.  So whatever mechanism we use to handle multiple group categories needs to be easily human-editable.

**Comment by edwery** [comment]
I think the export/import ought to use JSON, as its a common format - but there are tools which convert JSON to CSV and vice versa. We should probably have an export which just serializes JSON, then a further step which converts JSON to CSV - but as a start - since we currently export CSV, we should find out exactly what the tools produce when asked to convert a JSON array to CSV, and reproduce that for our arrays - so if we ever decide to switch to JSON export, it would be easy to do.

**Comment by mike99christie** [comment]
Although I generally support th idea of using JSON for APIs between systems, I think in this case we are really aiming at end users who will be able to edit spreadsheet files. using csv files is a practical lowest common denominator here.
I favour using vertical bar | as separator between multiple items in an entry and suppose that we will trim leading and trailing white space on both import and export.

**Comment by nicktrosnant** [comment]
Released in plugin version 1.6.2
